libsolidity/analysis/ReferencesResolver.cpp

```diff
@@ -36,9 +36,12 @@
 #include <boost/range/adaptor/transformed.hpp>
 
 using namespace std;
-using namespace dev;
-using namespace dev::solidity;
+using namespace langutil;
 
+namespace dev
```

Comment:
Usually we do not have namespaces in .cpp files, but I guess this is to avoid a name clash, right?